hbw/app/javascript/packs/hbw/components/form/file_upload.js.jsx

```diff
@@ -55,26 +58,50 @@ modulejs.define('HBWFormFileUpload', ['React'], (React) => {
     render () {
       const { translate, translateBP } = this.context;
       const {
-        name, params, hidden, fileListPresent, task
+        name, params, hidden, fileListNames, task
       } = this.props;
 
-      const { files } = this.state;
+      const { files, error } = this.state;
 
       const showFileInput = params.multiple || files.length === 0;
 
       const hiddenValue = JSON.stringify({ files });
       const cssClass = cx('hbw-file-upload', params.css_class, { hidden });
-      const errorMessage = translate('errors.file_list_field_required');
-      const errorMessageCss = cx('alert', 'alert-danger', { hidden: fileListPresent });
+
+      const errMessageParamEmpty = translate(
+        'errors.file_list_name_empty',
+        { field_name: name }
+      );
+
+      const errMessageParamNotMatched = translate(
+        'errors.file_list_name_not_matched',
+        { file_list_name: params.file_list_name }
+      );
+
+      const errMessageCss = 'alert alert-danger';
+
+      const getErrMessage = () => {
+        if (typeof params.file_list_name === 'undefined') {
+          return null;
```

Comment:
this is the `else` branch

Reply:
no

Reply:
If we get rid of this branch, we will fall to `errMessageParamNotMatched` branch in case of `'file_list_name' not in  params `